Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.8-slim
```

Comment:
Add an Environment variable as : `PYTHONUNBUFFERED=1` by adding:

```
ENV PYTHONUNBUFFERED=1
```

In the `Dockerfile`

So, python sends print and log statements directly to `stdout`. If we did not set this, we would not see logs from our container because they would be sent to a buffer.

Reply:
Before L#1 or after L#1?

Reply:
After L#1 as we're setting the environment variable in the Image. :)